docs/topics/api/addons.rst

```diff
@@ -503,7 +503,11 @@ Version Detail
 
 This endpoint allows you to fetch a single version belonging to a specific add-on.
 
-.. http:get:: /api/v5/addons/addon/(int:addon_id|string:addon_slug|string:addon_guid)/versions/(int:id)/
+    .. note::
+        Version numbers without a dot (``.``) character are not supported in
```

Comment:
Why are we allowing the `v` prefix to force the param to be recognized as a version number in other endpoints, but not in this endpoint?

Reply:
Actually, looking at the code and tests, it looks like we *do* allow it here as well, so perhaps this note needs to be updated to explain that.

Reply:
Yeah I should have updated it as well.